docs/BigQueryExecute-action.md

```diff
@@ -0,0 +1,45 @@
+# Google Cloud BigQuery Execute
+
+Description
+-----------
+This plugin executes a BigQuery SQL query.
+BigQuery is Google's serverless, highly scalable, enterprise data warehouse.
+
+Credentials
+-----------
+If the plugin is run on a Google Cloud Dataproc cluster, the service account key does not need to be
+provided and can be set to 'auto-detect'.
+Credentials will be automatically read from the cluster environment.
+
+If the plugin is not run on a Dataproc cluster, the path to a service account key must be provided.
+The service account key can be found on the Dashboard in the Cloud Platform Console.
+Make sure the account key has permission to access BigQuery.
+The service account key file needs to be available on every node in your cluster and
+must be readable by all users running the job.
+
+Properties
+----------
+**Project ID**: Google Cloud Project ID, which uniquely identifies a project.
+It can be found on the Dashboard in the Google Cloud Platform Console.
+
+**SQL**: The SQL command to execute.
+
+**Legacy**: Specified whether to use BigQuery's legacy SQL dialect for this query. By default this property is
+set to 'false'. If set to 'false', the query will use BigQuery's Standard SQL.
```

Comment:
It's better to have a 'Dialect' property with values 'Legacy' and 'Standard', with it defaulting to Standard. This is more extensible in case a new dialect is ever added, and also makes it easier to understand.